src/app/kvm/views/LXDClusterDetails/LXDClusterHosts/LXDClusterHosts.tsx

```diff
@@ -59,7 +63,7 @@ const LXDClusterHosts = ({
   );
 
   return (
-    <>
+    <div aria-label={Label.Title}>
```

Comment:
is there a reason why this couldn't be a `section`? I don't think we have any styles attached to the `section` itself?